utils/test/Is_LSMB_running.sh

```diff
@@ -13,6 +13,20 @@ if [[ -z $EX_NOHOST ]]; then
     exit 99;
 fi
 
+
+# base url for lsmb defaults to http://localhost:5001
+# this can be overriden by setting environment variable $LSMB_BASE_URL
+lsmb_base_url=${LSMB_BASE_URL:-http://localhost:5001}
```

Comment:
Unfortunately `$LSMB_BASE_URL` is actually NOT the correct variable to use.
That points at port 5001 which is the HTTPD port (and URL) for travis.yml
For the docker container it is correct though.

This test is specifically testing that STARMAN/PLACK is running and that is on port 5000 by default on travis.
For docker it's on 5762.

Currently there is no variable declared in travis.yml that defines the starman port.

I believe we need to declare a new `$STARMAN_BASE_URL` in both `travis.yml` and the docker container `start,sh` that points to the starman process in both cases, leaving `$LSMB_BASE_URL` to point at the httpd

Reply:
I propose to implement a new environment variable $PSGI_BASE_URL tomorrow (will sleep on it tonight).

When testing using the ledgersmb-dev-lsmb docker container, $PSGI_BASE_URL will be the same as $LSMB_BASE_URL - both will point to Starman.

When testing using Travis, $PSGI_BASE_URL will point to plack, while $LSMB_BASE_URL will point to the nginx proxy.

Positive feedback received in chat, but happy to hear any other thoughts:
https://matrix.to/#/!qyoLumPqusaXqFJNyK:matrix.org/$1513196388405391qbWsS:matrix.org

Reply:
Agreed with the proposal